uk.co.compucorp.civicrm.hrleaveandabsences/scss/mixins/_gradients.scss

```diff
@@ -1,3 +1,3 @@
 @mixin striped-background($stripe-color) {
-  background: repeating-linear-gradient(45deg, $stripe-color, $stripe-color 5px, transparent 5px, transparent 8px);
+  background: repeating-linear-gradient(45deg, $stripe-color, $stripe-color 2px, transparent 2px, transparent 7px);
```

Comment:
Make sure this mixin is not used anywhere else, which would result in unintentional changes.

Reply:
I wrote this mixin, it is ONLY used here:

![image](https://user-images.githubusercontent.com/3973243/40058975-b9bf8876-584a-11e8-9eca-b3bc24be52d4.png)